File: maximum-manuel.cs
using System;

class Program {
    static void Main() {
        Console.Write("Entrez des nombres séparés par des virgules: ");
        string str = Console.ReadLine();
        string[] strArray = str.Split(',');
        int[] array = Array.ConvertAll(strArray, int.Parse);
        
        if (array.Length == 0) {
            Console.WriteLine("Le tableau est vide");
            return;
        }

        TrouverMaximum(array);
    }

    static void TrouverMaximum(int[] array) {
        int maximum = array[0];

        for (int i = 0; i < array.Length; i++) {
            if (array[i] > maximum) {
                maximum = array[i];
            }
        }

        Console.WriteLine("Le plus grand nombre du tableau est " + maximum);
    }
}

File: node-project/maximum-manuel.js
const readline = require('readline').createInterface({
    input: process.stdin,
    output: process.stdout
});

readline.question("Entrez des nombres séparés par des virgules: ", function(str) {
    let strArray = str.split(',');
    let array = strArray.map(Number);

    if (array.length === 0) {
        console.log("Le tableau est vide");
        readline.close();
        return;
    }

    let maximum = array[0];

    for (let i = 0; i < array.length; i++) {
        if (array[i] > maximum) {
            maximum = array[i];
        }
    }

    console.log("Le plus grand nombre du tableau est ", maximum);
    readline.close();
});